templates/room.html

```diff
@@ -907,19 +967,88 @@
       orderList.splice(orderList.indexOf(target),1);
       return orderList;
     }
+    
+    
+    //returns boolean that tells you if it's your turn or not.
+    function isMyTurn(){
+        return currentTurn.turnUser === username;
+    }
+    
+    //increments turn class. Should be used after a move.
+    function nextTurn(){
+        
+        
+        playerList = getAllPlayers();      
+       
+        //change turn user   
+        if(playerList.indexOf(currentTurn.turnUser) + 1 === playerList.length){
+            currentTurn.turnUser = playerList[0]; //wrap around to beginning of array
+        }
+        else{                    
+            currentTurn.turnUser = playerList[playerList.indexOf(currentTurn.turnUser)+1];
+        }
+        
+        currentTurn.moved = false;
+        currentTurn.turnNum++;
+        sendMessageFromSystem('Turn: ' + currentTurn.turnNum,'all');
+        sendTurnJson();
+        sendMessageFromSystem(playerList,'all');
+    }
+    
+    function endTurn(){
+        if(currentTurn.turnUser === username){
```

Comment:
in this case, i will use single line if statement in javascript in order to increase the readability
return (currentTurn.turnUser === username) ? nextTurn() : alert('It is not your turn!');